src/mat_pixel.cpp

```diff
@@ -2555,8 +2555,14 @@ Mat Mat::from_pixels_resize(const unsigned char* pixels, int type, int w, int h,
     else if (type_from == PIXEL_RGBA || type_from == PIXEL_BGRA)
     {
         return Mat::from_pixels_resize(pixels, type, w, h, w * 4, target_width, target_height, allocator);
+    }else if(type_from==PIXEL_YUV420SP){
```

Comment:
be consistent with formatting from previous else if blocks in this file, in terms of use of white space and end of line characters (curly braces on separate lines)